src/shared/proto/robot_component_status.proto

```diff
@@ -0,0 +1,176 @@
+syntax = "proto2";
+
+/* Data about the status of the break beam */
+message BreakBeamStatus
+{
+    // Whether or not the ball is interrupting the robot’s laser beam.
+    required bool ball_in_beam = 1;
+
+    // The reading of the robot’s laser sensor.
+    required double break_beam_reading = 2;
+
+    // The maximum full-scale deflection (maximum possible reading) of the laser sensor.
+    required double break_beam_scale = 3;
+}
+
+/* Data about the status of the chipper and kicker */
+message ChipperKickerStatus
+{
+    // Whether or not the robot’s capacitor is charged enough to kick the ball.
+    required bool capacitor_charged_enough_to_kick_ball = 1;
+
+    /* Frankie Robot Fields */
+    // Indicates the time elapsed since chipper was last fired in ms
+    // UINT64_MAX indicates chipper was never fired
+    optional uint64 time_since_chipper_fired = 2;
+
+    // Indicates the time elapsed since kicker was last fired in ms
+    // UINT64_MAX indicates kicker was never fired
+    optional uint64 time_since_kicker_fired = 3;
+
+    /* 2014 Robot Fields */
+    // The maximum possible kick speed, in m/s.
+    optional double kick_speed_max = 4;
+
+    // The kick resolution for HScale.
+    optional double kick_speed_resolution = 5;
+
+    // The maximum possible chip distance, in m.
+    optional double chip_distance_max = 6;
```

Comment:
I don't think this field is useful and we can remove it. At best this should be a constant somewhere else.